### PR TITLE
chore: bump dependencies

### PR DESCRIPTION
**Pull request by nulltoken**
**Checklist**

- [ ] Tests added / updated
- [ ] Docs added / updated

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by nulltoken** [comment]
@philsturgeon This embeds the update to the latest stable Spectral.
Once this is merged, I'm going to publish a minor update to the extension and get it published.

**Comment by nulltoken** [comment]
@philsturgeon Don't merge it yet. I'm going to update it tomorrow with latest versions of dependencies

**Comment by nulltoken** [comment]
> @philsturgeon Don't merge it yet. I'm going to update it tomorrow with latest versions of dependencies

@philsturgeon Updated and rebased. Ready for review.

**Review by philsturgeon — APPROVED** [review]

**Pull request merged**
